task/src/lastpencil/PencilGame.java

```diff
@@ -3,4 +3,113 @@
 import java.util.Random;
 import java.util.Scanner;
 
-//Class for Pencil Game
\ No newline at end of file
+public class PencilGame {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final Random random = new Random();
+    public static int pencilsTaken;
+
+    public static int getNumberOfPencils() {
+        int numberOfPencils;
+        do {
+            System.out.println("How many pencils would you like to use:");
+            numberOfPencils = positiveValidation();
+            if (numberOfPencils <= 0) {
+                System.out.println("The number of pencils should be positive");
+            }
+        } while (numberOfPencils <= 0);
+
+        return numberOfPencils;
+    }
+
+    public static int positiveValidation() {
+        while (true) {
+            try {
+                return Integer.parseInt(scanner.nextLine());
+            } catch (NumberFormatException e) {
+                System.out.println("The number of pencils should be numeric.");
+            }
+        }
+    }
+
+    public static String getPlayer() {
+        String firstPlayerName;
+        do {
+            System.out.println("Who will be the first (John, Jack):");
+            firstPlayerName = scanner.nextLine();
+            if (!firstPlayerName.equals("John") && !firstPlayerName.equals("Jack")) {
+                System.out.println("Choose between 'John' and 'Jack'");
+            }
+        } while (!firstPlayerName.equals("John") && !firstPlayerName.equals("Jack"));
+        return firstPlayerName;
+    }
+
+    public void printOutPencils(int numberOfPencils, String firstPlayerName) {
```

Comment:
This is almost the only method that is not static... is there a reason for that (or conversely: why are almost all of the other methods static)?